### ISSUE TITLE
[BUG] qbcore crafting option

### ISSUE THREAD

**Issue by lenclint** [qbcore-framework/qb-core#1147]
### Summary

Im having trouble finding out how to craft items. I went to the toolbox in the scrapyard to craft items but the prompt to use the toolbench is not showing. Im sure i have the right files

### Reproduction

opened up my server and teleported to scrapyard to craft

### Expected behavior

'E' prompt to appear on toolbox

### Actual behavior

no prompted appeared

### Last Updated

today

### Custom Resources

 gigz_weapon_room2,  gunstore_fivem, online-interiors-master, pd_sandy, qt-library, qt-crafting-v2

### Resource Rename

no

**Comment by brandon1808** [comment]
Are you getting any erros.

**Comment by minijoker2o0** [comment]
yes the item needs to be added to the items.lua in shared folder, what i want to know is how can we make it usable with out having to have it in your inventory and use it, anyone know where the lines are to stop that from happening so we can spawn the prop and place it in a secret location to stop people farming the craft from the get go.
